### ISSUE TITLE
[Feature Request] navigate through folders using arrow keys

### ISSUE THREAD

**Issue by b-mq** [oliverschwendener/ueli#477]
It would be great to navigate through folders using ← or → arrow keys.
E.g. I searched for the Downloads folder and have it selected in Ueli, by pressing the ← I would go UP one folder and by pressing → I would go into the selected folder.

**Comment by oliverschwendener** [comment]
I don't think we should use the arrow keys for folder navigation because they are used to move the cursor in the input field. Maybe in combination with Ctrl.

FYI: currently you can use <kbd>Tab</kbd> to navigate into the folder.

**Comment by bastiMQ** [comment]
I personally do not use the arrow keys a lot to navigate the cursor, but sure, maybe another key combination would work as well.
Maybe later you can also provide the option to remap keys in order to use your personal preferred method.

PS: Thanks for the tip with the Tab key

**Comment by oliverschwendener** [comment]
See #325
